yaks-html/yaks-html.gemspec

```diff
@@ -12,10 +13,10 @@ Gem::Specification.new do |gem|
   gem.homepage    = 'https://github.com/plexus/yaks'
   gem.license     = 'MIT'
 
-  gem.require_paths    = %w[lib]
-  gem.files            = `git ls-files`.split($/)
+  gem.require_paths    = %w(lib)
+  gem.files            = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
   gem.test_files       = gem.files.grep(/^spec/)
-  gem.extra_rdoc_files = %w[README.md]
+  gem.extra_rdoc_files = %w(README.md)
```

Comment:
here I have a strong opinion on. You're writing an array, `[` and `]` is more clear